R/dfm_weight.R

```diff
@@ -241,6 +241,23 @@ dfm_smooth.dfm <- function(x, smoothing = 1) {
 #' feature occurs more than a given frequency threshold.  (The default threshold
 #' is  zero, meaning that any feature occurring at least once in a document will
 #' be counted.)
+#' @param x a [dfm]
```

Comment:
I had to restore these because although the argument names are the same as in `dfm_weight()`, the documentation entries are different (except `threshold`).